compiler/src/iree/compiler/Codegen/SPIRV/SPIRVVectorizeLoadStore.cpp

```diff
@@ -32,6 +32,8 @@
 
 #define DEBUG_TYPE "iree-spirv-vectorize-load-store"
 
+#define DO_NOTHING [](OpBuilder &b, Location loc) { return ValueRange{}; }
```

Comment:
You can make it a regular function. Also this will generate unused argument warnings in this form.
```suggestion
void DoNothing(OpBuilder &, Location) { }
```

Reply:
yeah gcc just told me what I was doing before was a bad idea